hosted-ce/etc/osg/image-config.d/30-remote-site-setup.sh

```diff
@@ -111,6 +111,11 @@ setup_endpoints_ini () {
     # This relies on the tarball-install/23 format (instead of tarball-install/23-main)
     # on the OSG Yum repo
     osg_ver=$(rpm -q osg-release --qf "%{VERSION}")
+    # HACK: OSG 23 does not support EL7, force the OSG 3.6 tarball
+    # until we get guidance on EL7 support for remote sites (SOFTWARE-5880)
+    if [[ $osg_ver == "23" ]] && [[ $remote_os_major_ver == "7" ]]; then
+        osg_ver="3.6"
+    fi
```

Comment:
```suggestion
    if [[ $remote_os_major_ver == "7" ]]; then
        osg_ver="3.6"
    fi
```
while I _hope_ we'll have stopped sending el7 tarballs by the time OSG 24 is released, I can't be certain about it.

Reply:
Sigh you're right